ttnn/cpp/ttnn/operations/data_movement/concat/concat.cpp

```diff
@@ -86,7 +88,9 @@ MassagedConcat build_unsqueeze_concat(int input_rank, const MemoryConfig& output
         }});
 }
 
-MassagedConcat build_untilize_rm_retilize_concat(uint8_t queue_id, const MemoryConfig& output_memory_config) {
+MassagedConcat build_untilize_rm_retilize_concat(uint8_t queue_id,
+                                                 const MemoryConfig& output_memory_config,
+                                                 ttnn::SimpleShape logical_output_shape) {
```

Comment:
Can be const ref?

Reply:
The SimpleShape. Good point, I'll make a note to do that 🫡